### PR TITLE
[3.2] Upgrade to Quarkus CXF 2.2.5 and Camel Quarkus 3.2.1

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
07473ae: 

* Disabled one of the failing tests (`MtomTest`). It is sending many rather large requests, so read timeouts may happen on the underpowered GA hosts. 
* The other one `CxfSoapMtomTest` should actually pass. It sends just a small (38 kB) image.

**Comment by ppalaga** [comment]
36e277d: regenerated

**Comment by ppalaga** [comment]
Is there any chance to get this merged and released any time soon? Our users are waiting for these upgrades.

**Comment by davsclaus** [comment]
From ASF this is concerning and unacceptable that Camel end users cannot get patch released in their hands.
This PR has been sitting idle for 3 weeks and we have 4.0.2 on the way soon, then is it the same "delay" next time :(

File: generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-mtom/pom.xml [hunk 1]
```diff
@@ -10,6 +10,9 @@
   </parent>
   <artifactId>quarkus-cxf-integration-test-mtom</artifactId>
   <name>Quarkus Platform - CXF - Integration Tests - quarkus-cxf-integration-test-mtom</name>
+  <properties>
+    <maven.test.skip>true</maven.test.skip>
```
Comment (rsvoboda) [inline generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-mtom/pom.xml:14]: mtom regression ? @vkasala fyi

Comment (ppalaga) [reply generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-mtom/pom.xml:14]: Not really a regression in MTOM, it is rather that the test has changed recently and became longer-running which makes it more likely to fail on GHA infra - see https://github.com/quarkiverse/quarkus-cxf/issues/1067
In the main platform branch, we shortened the default duration time already and the test is not disabled anymore.

**Comment by ppalaga** [comment]
Thanks for merging @aloubyansky!

**Pull request merged**